plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,11 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// issue: https://github.com/kubernetes/kubernetes/issues/43845
+	// for backward compatibility if user set the value in policy file, then we use it
```

Comment:
@bsalamat Backward scenario is use don't set the value in policy file, it will use the value from command line when create the scheduler. here the logic is if we set the value in policy file, we should use the value from new location.